test/e2e-v2/cases/mysql/prometheus-mysql-exporter/otel-collector-config.yaml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+receivers:
+  prometheus:
+    config: 
+     scrape_configs:
+       - job_name: 'mysql-monitoring'
+         scrape_interval: 10s
+         static_configs:
+           - targets: ['mysql-service:9104']
+             labels: 
+               host_name: showcase
+processors:
+  batch:
+
+exporters:
+  logging:
+    loglevel: debug
+  opencensus:
+    endpoint: oap:11800
+    tls:
+      insecure: true
+service:
+  pipelines:
+    metrics:
+      receivers:
+      - prometheus
+      processors:
+      - batch
+      exporters:
+      - logging
```

Comment:
What is logging exporter for?

Reply:
Just export to cmd. It is not necessary, just for user to see the metrics.
Need I remove this?

Reply:
> ```
> commands_update_rate
> commands_insert_rate
> commands_delete_rate
> ```
> 
> You could write a python or any simple app you prefer. I think when your codes access MySQL, those metrics could have values.

I thought about it, but I am comfused about which part should I to execute my code in the e2e.
I have tested a few methods, but they all fail in the end.

Reply:
If the logs are not too many, it is fine to keep.

Reply:
There are indeed a lot of logs, I should remove them.